src/com/cft/contactmerge/FirstNameMatcher.java

```diff
@@ -0,0 +1,42 @@
+/* Created by James Page on May 8th, 2018
+ * Contains the order in which the first names will be considered a match or not...
+ * Exact Match (and matched with punctuations), matched with hyphenated parts, matched with apostrophe, and matched with initial.
+ */
+
+package com.cft.contactmerge;
+
+import java.util.ArrayList;
+
+public abstract class FirstNameMatcher extends NameMatchLogic {
+    public static AnswerType getFirstNameMatchResult(ArrayList<String> contactOneFirstName, ArrayList<String> contactTwoFirstName){
```

Comment:
Since these are collections, should be "contactOneFirstNames" and "contactTwoFirstNames"

Reply:
okay